src/core/manager.h

```diff
@@ -511,6 +511,9 @@ struct Manager {
         int executor_fd;
 
         unsigned soft_reboots_count;
+
+        /* Ambient capability set for use with *some* forked processes */
```

Comment:
"*some*" is bit unspecific.

Just say "Original ambient capabilities when we were initialized", because that's what it is.

Reply:
DONE